features/users.feature

```diff
@@ -16,53 +16,28 @@ Feature: Users
     Then I should see "Security Notification"
     And I should see "This is a U.S. General Services Administration Federal Government computer system"
 
-  # to be updated in SRCH-862 for login.gov
+  # to be updated in SRCH-1237 for login.gov
   @wip
   @javascript
   Scenario: Registering as a new affiliate user who is a government employee or contractor with .gov email address
     Given I am on the sign up page
     When I fill in the following:
       | Your full name | Lorem Ipsum            |
       | Email          | lorem.ipsum@agency.gov |
-      | Password       | short                  |
     And I press "Sign up"
-    Then I should see "Your new password must be different from your old password. Passwords must contain a minimum of eight (8) characters and include a combination of letters, numbers, and special characters. Passwords are good for 90 days."
     And I should see "Federal government agency can't be blank"
     When I fill in the following:
       | Your full name            | Lorem Ipsum            |
       | Email                     | lorem.ipsum@agency.gov |
-      | Password                  | test1234!              |
       | Federal government agency | Agency                 |
     And I press "Sign up"
     Then I should be on the user account page
     When I sign out
     Then I should be on the login page
-    Given a clear email queue
-    Then the "Email" field should contain "lorem.ipsum@agency.gov"
-    When I fill in the following:
-      | Password | test1234!            |
-    And I press "Login"
-    Then I should see "Thank you for verifying your email."
-    And I should be on the user account page
     And "lorem.ipsum@agency.gov" should receive an email
     When I open the email
     Then I should see "Welcome to Search.gov" in the email subject
 
-  # to be updated in SRCH-862 for login.gov
-  @wip
-  Scenario: Registering as a new affiliate user with .gov email address and trying to add new site without email verification
-    Given I am on the sign up page
-    When I fill in the following:
-      | Email                     | lorem.ipsum@agency.gov |
-      | Your full name            | Lorem Ipsum            |
-      | Password                  | test1234!              |
-      | Federal government agency | Agency                 |
-    And I press "Sign up"
-    Then I should be on the user account page
-    When I follow "Add Site"
-    Then I should be on the user account page
-    And I should see "Your email address has not been verified. Please check your inbox so we may verify your email address."
-
```

Comment:
I took this one out because its testing Authentication and email verification which login.gov does now.

Reply:
👍